code/datums/traits/negative.dm

```diff
@@ -91,7 +91,7 @@
 	medical_record_text = "Patient has a tumor in their brain that is slowly driving them to brain death."
 
 /datum/trait/brainproblems/on_process()
-	trait_holder.adjustBrainLoss(0.2)
+	trait_holder.adjustBrainLoss(0.2, 0)
```

Comment:
Can you go over 

https://github.com/tgstation/tgstation/blob/8e77f937a38782ca1410ebeb499d4e094c6df1d8/code/modules/mob/living/brain/brain_item.dm#L149

I think your fix is fixing the local issue rather than the root since it doesn't do brain maths if you set it to 0.

Reply:
I didn't know it worked like that, sorry

Reply:
it's not suppose to which is the problem.

Reply:
I'm looking into a better fix now